modules/plottinggl/src/plotters/persistencediagramplotgl.cpp

```diff
@@ -431,7 +436,21 @@ void PersistenceDiagramPlotGL::setIndexColumn(
     }
 }
 
-void PersistenceDiagramPlotGL::renderAxis(const size2_t &dims) {
+void PersistenceDiagramPlotGL::setSelectedIndices(const std::unordered_set<size_t>& indices) {
```

Comment:
Should we copy this or use shared pointers?

Reply:
Brushing and Linking ports work with explicit sets...

https://github.com/inviwo/inviwo/blob/0e581cfda4ea3d2edc1f4c98a6e8d6256cc298a9/modules/brushingandlinking/include/modules/brushingandlinking/ports/brushingandlinkingports.h#L54-L65